src/explore-education-statistics-admin/src/pages/admin-dashboard/components/__tests__/MethodologySummary.test.tsx

```diff
@@ -345,6 +351,125 @@ describe('MethodologySummary', () => {
       ).toBeInTheDocument();
     });
 
+    test('the edit button is shown when a user can approve the methodology', () => {
```

Comment:
Might want to say 'edit link' instead of button to be a little more specific

Reply:
Thanks, `edit button` renamed to `edit link`